src/utils/usage-analytics.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+import cli from 'cli-ux';
+import * as fs from 'fs';
+import * as path from 'path';
+
+export async function usageAnalytics(projectDir: string, agreeToAnalytics: boolean) {
+  if (agreeToAnalytics === false) {
+    return;
+  }
+  if (agreeToAnalytics || (await cli.confirm('Do you want to provide anonymous usage analytics to help us improve the SDK? (y|n)'))) {
```

Comment:
```suggestion
  if (agreeToAnalytics !== false || (await cli.confirm('Do you want to provide anonymous usage analytics to help us improve the SDK? (y|n)'))) {
```

Then you could remove the first if.

Reply:
This is intentional. The suggestion would change the behavior of the if statement